programs/programs/policyengine/engines.py

```
@@ -45,30 +45,41 @@ def members(self, unit, sub_unit):
+        "audience": "https://household.api.policyengine.org",
+    }
+    try:
+        res = requests.post(_pe_token_url, json=payload, timeout=(5, 30))
+        res.raise_for_status()
+        data = res.json()
+        token = data["access_token"]
+        expires_in = int(data.get("expires_in", 86400))
+    except requests.RequestException as e:
+        raise RuntimeError(f"Failed to fetch PolicyEngine bearer token: {e}") from e
+    except (ValueError, KeyError) as e:
+        raise RuntimeError(f"Invalid response from PolicyEngine token endpoint") from e
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)